main/getting-started/start-a-project.md

```diff
@@ -4,13 +4,10 @@
 The Agoric platform is at the beta stage. It is in the process of being formally evaluated for vulnerabilities, and is undergoing security treatment and hardening to support mainnet-1 launch. Do not use for production purposes. 
 :::
 
-Now that you have [installed the Agoric SDK](/getting-started/before-using-agoric.md),
-let's try out your first Agoric _Dapp_ (decentralized application).
-
-We'll be running **three terminal windows**, and
-[dapp template initialization](#initialize-demo-from-dapp-template) in the first
-window will create their shared working directory.
+Do not proceed unless you've [installed the Agoric SDK](/getting-started/before-using-agoric.md).
```

Comment:
Starting the page with "Do not" seems sort of jarring. Any particular reason for that change?

Reply:
I actually wanted sort of jarring... this entire page is a non-starter unless they've got `agoric` available from their terminal and that seems to be a point of friction for many, but of course we can soften the language.

Reply:
But if they follow the docs in the normal order, they'll have `agoric` installed, yes?

In any case, I much prefer that we phrase it positively; for example: "Use `agoric --version` to double-check that you are done [installing the Agoric SDK](...)". If people are still tripping over it, I suggest we use a call-out box rather than starting with "Do not ...".

Reply:
more positive it is now! <(°.°)>